README.md

```diff
@@ -0,0 +1,33 @@
+# Scripts and workflows to build and update the CFDE portal content registry
+
+## Dependencies
+
+You'll need a modern Python (3.8+) together with snakemake and
+cfde-deriva. If you conda/mamba, you can do that with the below:
+
+```
+mamba create -n content-reg -c bioconda -y snakemake-minimal
+conda activate content-reg
+pip install git+https://github.com/nih-cfde/cfde-deriva.git
+```
+
+Alternatively, running
+```
+pip install git+https://github.com/nih-cfde/cfde-deriva.git snakemake
+```
+should get you there.
+
+## Quickstart - building
```

Comment:
Need to add steps for cloning the repo (and maybe checking out a branch)

Reply:
I'll put the clone/fetch stuff over in #2, thanks!

In re pushing to app-dev and updating app-dev, this is only something that the CFDE-CC team will have permissions to do so we will document it internally. (There are multiple reasons for this but the simplest reason is that we can't limit it to app-dev - if you have this power on app-dev, you have this power on the live server/full portal, too!)